.github/FUNDING.yml

```diff
@@ -8,5 +8,5 @@ tidelift: # Replace with a single Tidelift platform-name/package-name e.g., npm/
 community_bridge: # Replace with a single Community Bridge project-name e.g., cloud-foundry
 liberapay: # Replace with a single Liberapay username
 issuehunt: # Replace with a single IssueHunt username
-otechie: # Replace with a single Otechie username
```

Comment:
What is this?

Reply:
I have reverted it, resolving the thread